.snippets/text/builders/ethereum/precompiles/ux/erc20/erc-20-interface.md

```diff
@@ -0,0 +1,19 @@
+The interface includes the following functions:
```

Comment:
This is only used in one place (`interact.md`) so we should delete this here and just put it in that file

Side note, the snippet name used to be: `erc20-interface.md`, so you'll have to make sure you search for that when going to replace it